pyupgrade.py

```diff
@@ -1092,6 +1104,13 @@ def generic_visit(self, node):
         super(FindPy3Plus, self).generic_visit(node)
 
 
+def _is_utf8_codec(encoding):
+    try:
+        return codecs.lookup(encoding).name == 'utf-8'
+    except LookupError:
+        return False
```

Comment:
I tend to put helpers _higher_ than their callers in the file, nbd though

Reply:
sure